src/evict.c

```diff
@@ -210,7 +210,7 @@ int evictionPoolPopulate(serverDb *db, kvstore *samplekvs, struct evictionPoolEn
         /* Try to reuse the cached SDS string allocated in the pool entry,
          * because allocating and deallocating this object is costly
          * (according to the profiler, not my fantasy. Remember:
-         * premature optimization bla bla bla. */
+         * premature optimization blah, blah, blah. */
```

Comment:
`blah` is a word, and I'm proposing to use it consistently both in comments (as here) and in code. The commas here are proper for English...

I'm happy to drop anything as requested...